backend/LexData/Entities/UserEntityConfiguration.cs

```diff
@@ -17,6 +18,8 @@ public override void Configure(EntityTypeBuilder<User> builder)
         builder.HasIndex(u => u.Username).IsUnique();
         builder.Property(u => u.Email).UseCollation(LexBoxDbContext.CaseInsensitiveCollation);
         builder.HasIndex(u => u.Email).IsUnique();
+        builder.Property(u => u.FeatureFlags).HasDefaultValue(new List<FeatureFlag>()); // Will this create the *same* list object for all rows, thus creating a bug?
```

Comment:
probably not? but you can use the callback based version to correct that `.HasDefaultValue(() => new List<FeatureFlag>())`

Reply:
I looked for a callback and didn't see it in Intellisense or the API documentation. I'll look again.

Reply:
Using a callback `() => new List<FeatureFlag>()` produces "Cannot set default value 'System.Func\`1[System.Collections.Generic.List\`1[LexCore.FeatureFlag]]' of type 'System.Func\`1[System.Collections.Generic.List\`1[LexCore.FeatureFlag]]' on property 'FeatureFlags' of type 'System.Collections.Generic.List\`1[LexCore.FeatureFlag]' in entity type 'User'". So as I thought from the docs, it's trying to set the callback's Func object as the value of the column, rather than using the callback. There's a version with a factory object, but that's way more boilerplate than is worth using here, since the simple `new List<FeatureFlag>` solution works and appears to not exhibit the buggy behavior I was worrying about.

Reply:
ahh ok, yeah that makes sense, there wasn't a callback version it was just accepting the func as an object lol